test/test_terms.py

```diff
@@ -45,6 +45,11 @@
 np.seterr(all='ignore')
 
 
+# have to make this deterministic for pytest collection to work
```

Comment:
Could you explain the type of error you saw, so I can keep an eye out for similar errors in the future?

Reply:
`pytest-xdist` seems to require all workers to have identical test cases that appear in the same order.  Some test cases in this file are generated by looping over the keys of `REDUCE_OP_TO_NUMERIC`, which is a dictionary.  If you revert this change and attempt to run the tests in Python 3.5 with multiple workers (`pytest -n auto`), test collection will nondeterministically fail because workers may not agree.